e2e/cypress/integration/channel_sidebar/category_collapsing_unread_filter_spec.js

```diff
@@ -37,6 +37,9 @@ describe('Channel sidebar', () => {
     });
 
     it('should display collapsed state when collapsed', () => {
+        // # Close "What's new" modal
+        cy.uiCloseWhatsNewModal();
```

Comment:
I'm good as is but better if this is placed in the before hook so each test can be run with `.only` when have to in the future.

Reply:
Reason is page is loaded in beforeEach and modal only appear only once. That would require restructuring the whole test